src/jdk.management/share/classes/com/sun/management/internal/HotSpotDiagnostic.java

```diff
@@ -53,20 +49,11 @@ public HotSpotDiagnostic() {
     public void dumpHeap(String outputFile, boolean live) throws IOException {
 
         String propertyName = "jdk.management.heapdump.allowAnyFileSuffix";
-        PrivilegedAction<Boolean> pa = () -> Boolean.parseBoolean(System.getProperty(propertyName, "false"));
-        @SuppressWarnings("removal")
-        boolean allowAnyFileSuffix = AccessController.doPrivileged(pa);
+        boolean allowAnyFileSuffix = Boolean.parseBoolean(System.getProperty(propertyName, "false"));
```

Comment:
can be simplified to `Boolean.getBoolean(propertyName)`

Reply:
Yes, missed that.